### ISSUE TITLE
[Bug Report]: Library nested amqplib in NestJS

### ISSUE THREAD

**Issue by blaicer15** [farm-fe/farm#1520]
### Steps to reproduce

After following the instructions

```
success Installed "create-farm@0.0.10" with binaries:
      - create-farm

 ⚡ Welcome To Farm ! 

✔ Project name · backend-farm
✔ Select a framework: · NestJS - (https://nestjs.com/)

 ✔️ Template copied Successfully! 


 >  Initial Farm Project created successfully ✨ ✨ 

    cd backend-farm 

    yarn 

    yarn dev 

Done in 16.84s.
```

### the following error occurs:

```
yarn dev        
yarn run v1.22.22
$ farmup -w
[ Farmup ] Using config file at ~/backend-farm/farm.config.ts
[ Farmup ] [entry: ./src/main.ts] [format: esm] [target: node]
[ building ] ⠄ transform (1825) node_modules/object-hash/index.js                                                                                                                                                                             [ Farmup ] amqplib not found, it to be set external
[ Farmup ] amqplib not found, it to be set external
[ Farmup ] amqplib not found, it to be set external
[ Farmup ] Build completed in 923ms ⚡️FULL EXTREME! Resources emitted to ./dist.
[ Farmup:NestJs ] node:internal/modules/esm/resolve:853
  throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath(base), null);
        ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'amqplib' imported from ~/backend-farm/dist/NestJs.mjs_0012.fdaa606f.mjs
    at packageResolve (node:internal/modules/esm/resolve:853:9)
    at moduleResolve (node:internal/modules/esm/resolve:910:20)
    at defaultResolve (node:internal/modules/esm/resolve:1130:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:396:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:365:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:240:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:85:39)
    at link (node:internal/modules/esm/module_job:84:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v20.11.1

[ Farmup ] "NestJs" PID 22251 exit 1
```

### What is actually happening?

Does not locate the necessary library

### System Info

```Shell
System:
    OS: Linux 5.15 Linux Mint 21.3 (Virginia)
    CPU: (4) x64 Intel(R) Core(TM) i5-7400 CPU @ 3.00GHz
    Memory: 2.65 GB / 7.71 GB
    Container: Yes
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 20.11.1 - ~/.nvm/versions/node/v20.11.1/bin/node
    Yarn: 1.22.22 - /usr/bin/yarn
    npm: 10.5.2 - ~/.nvm/versions/node/v20.11.1/bin/npm
    bun: 1.0.30 - ~/.bun/bin/bun
  Browsers:
    Brave Browser: 125.1.66.118
    Chrome: 126.0.6478.126
  npmPackages:
    @farmfe/core: ^1.2.4 => 1.2.6
```

**Comment by ErKeLost** [comment]
Farm currently needs to install these packages manually. Importing farm under dynamic conditions has not yet been processed. I will take a look first.
https://github.com/farm-fe/farm/issues/1521 this one

**Comment by shulandmimi** [comment]
this seems to be a problem with farm external. the temporary solution is to download 'amqplib' and we will fix it later

**Comment by Crhis35** [comment]
I got this error also when trying to migrate an existing nestjs project:

Error: Can not load `node_modules/fsevents/fsevents.node`. Original error: 
None.

Potential Causes:
1. This kind of module is not supported, you may need plugins to support it.

<img width="760" alt="Screenshot 2024-07-06 at 8 23 02 AM" src="https://github.com/farm-fe/farm/assets/49347705/3a3d64eb-9378-420a-8bb1-235645017fef">

**Comment by shulandmimi** [comment]
@Crhis35 try set fsevents external, this issue has been fixed in #1527, but it will take some time to be released to npm according to our plan.

**Comment by ErKeLost** [comment]
fixed in 1.3.0
